parts/linux/cloud-init/artifacts/reconcile-private-hosts.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+SLEEP_SECONDS=15
```

Comment:
any chance we would need to adjust this value in future? consider make it a toggle in future, in necessary

Reply:
we don't expect to change this value. should be relatively stable.